pkg/controller/sparkapplication/controller.go

```diff
@@ -537,6 +543,7 @@ func (c *Controller) syncSparkApplication(key string) error {
 				// Successfully deleted driver or driver doesn't exist. Remove it from the Finalizer List.
 				appToUpdate = removeFinalizer(appToUpdate, sparkDriverRole)
 				glog.Infof("Submitting SparkApplication %s", key)
+				appToUpdate.Status.TerminationTime = metav1.NewTime(time.Unix(0, 0))
```

Comment:
You can simply do `appToUpdate.Status.TerminationTime = metav1.Time{}` to clear it.